PAO-Project/src/Utilizator/Student.java

```diff
@@ -0,0 +1,31 @@
+package Utilizator;
+
+public class Student extends Utilizator{
+    private int an_studiu;
+    private String specializare;
+
+    public Student(){}
+    public Student(int id, String nume, Contact info, Adresa adresa, int an_studiu, String specializare)
+    {
+        super(id,nume,info,adresa);
+        this.an_studiu = an_studiu;
+        this.specializare = specializare;
+
+    }
+
+    @Override
+    public void Date_Utilizator()
```

Comment:
[Fix] asta ar fi mers direct intr-un override de toString, nu vad nevoie unei metode extra

Reply:
Metoda Date_Utilizator este folosita pe post de get_All pentru a afisa toate datele clasei. Afiseaza atat cuvinte cat si valori numerice ce nu sunt convertite explicit in string si nu are return type.